src/test/java/teammates/test/cases/ui/InstructorEditStudentFeedbackSaveActionTest.java

```diff
@@ -510,10 +497,70 @@ private void testSubmitResponseForInvalidQuestion() throws Exception {
         };
         
         InstructorEditStudentFeedbackSaveAction a;
+        @SuppressWarnings("unused")
+        // unused but still needed to allow detection of exception
         RedirectResult r;
-        try{
+        
```

Comment:
These two are added to make the branch coverage more full, i.e. the visibility settings from testExecuteAndPostProcess()